src/components/Table/pageObject/index.js

```diff
@@ -0,0 +1,31 @@
+const PageHeader = require('../head/pageObject/index');
+
+/**
+ * Table page object class.
+ * @class
+ */
+class PageTable {
+    /**
+     * Create a new Table page object.
+     * @constructor
+     * @param {string} rootElement - The selector of the VerticalNavigation root element.
+     */
+    constructor(rootElement) {
+        this.rootElement = rootElement;
+    }
+
+    /**
+     * Returns a new Header page object of the element in item position.
+     * @method
+     * @param {number} itemPosition - The base 0 index of the header.
+     */
+    getItem(itemPosition) {
```

Comment:
we need to be more specific since in the future is likely that we need to get cells so:
```getHeaderItem```